src/presentational-components/shared/table-toolbar-view.js

```diff
@@ -0,0 +1,219 @@
+import React, { Fragment, useEffect, useState } from 'react';
+import propTypes from 'prop-types';
+import { Table, TableHeader, TableBody } from '@patternfly/react-table';
+import {
+  defaultSettings,
+  getCurrentPage,
+  getNewPage
+} from '../../helpers/shared/pagination';
+import { ListLoader } from './loader-placeholders';
+import { useIntl } from 'react-intl';
+import { Section } from '@redhat-cloud-services/frontend-components/components/cjs/Section';
+import { PrimaryToolbar } from '@redhat-cloud-services/frontend-components/components/cjs/PrimaryToolbar';
+
+export const TableToolbarView = ({
```

Comment:
I believe we don't really need most of these props. Can you reduce the list a bit?